.rubocop.yml

```diff
@@ -109,7 +119,7 @@ Style/Documentation:
     - !ruby/regexp /features\/.*.rb$/
 Style/DoubleNegation:
   Enabled: false
-Style/FileName:
+Style/Encoding:
```

Comment:
Can we just exclude problematic files?

Reply:
This cop flags files that contain the following magic comment because from Ruby 2.0+, UTF-8 is the default source file encoding:
```ruby
# encoding: UTF-8
```
So, IMO, we switch to `when_needed` if we're to enable the cop -- only enforce an encoding comment if there are non-ASCII chars

Reply:
Let's do `when_needed`, then!